specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json

```diff
@@ -1227,25 +1255,57 @@
         "Stopped"
       ],
       "x-ms-enum": {
-        "name": "SessionStatus",
-        "modelAsString": true
+        "name": "session_status",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Error",
+            "description": "The rendering session has encountered an error, and is unusable. This is a terminal state."
+          },
+          {
+            "value": "Expired",
+            "description": "The rendering session enters the 'Expired' state when it has been in the 'Ready' state longer than its lease time. This is a terminal state."
+          },
+          {
+            "value": "Starting",
+            "description": "The rendering session is starting, but not accepting incoming connections yet."
+          },
+          {
+            "value": "Ready",
+            "description": "The rendering session is ready for incoming connections."
+          },
+          {
+            "value": "Stopped",
+            "description": "The rendering session has been stopped with 'Stop Session' operation. This is a terminal state."
+          }
+        ]
       },
       "type": "string",
       "example": "Ready",
-      "description": "The status of the rendering session. Once the status reached the 'Ready' state it can be connected to. The terminal state is 'Stopped'."
+      "description": "The status of the rendering session."
     },
     "session_size": {
       "enum": [
         "Standard",
         "Premium"
       ],
       "x-ms-enum": {
-        "name": "SessionSize",
-        "modelAsString": true
+        "name": "session_size",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Standard",
+            "description": "Standard size has a maximum scene size of 20 million polygons."
```

Comment:
Rather than inlining limitations (20M polygons) explicitly here, I would propose referring to the docs, so this info does not get out-of-date:
"For specific limitations of respective server sizes, refer to this documentation: https://docs.microsoft.com/azure/remote-rendering/reference/vm-sizes"